examples/magento-graphcms/pages/checkout/added.tsx

```diff
@@ -77,15 +83,15 @@ function CheckoutAdded() {
         )}
 
         <Box gridArea='children'>
-          <Box sx={{ typography: 'h6' }}>
+          <Box sx={{ typography: 'h6' }} tabIndex={-1} ref={a11yFocusRef}>
```

Comment:
Focussing on a `tabIndex={-1}`?

Reply:
Yes, the H6 normally should not be focusable. (Thus -1) but when the user adds an item to his cart we do want to shift focus to the modal. (And in this case have the screenreader tell the user the action worked)